test/pythonml/test_pre_processing_flavors.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+import importlib
+import os
+import shutil
+import unittest
+from parameterized import parameterized
+import pandas
+import numpy as np
+from fixtures.unittest_baseclass import BaseUnitTest
+
+
+def load_test_train_targets_and_descriptors():
+    """
+    Loads a set of commonly used pickle files from an updated location in the context object
+    """
+    # import and roload settings to update the context object's path dir
+    import settings
+    importlib.reload(settings)
+    train_target = settings.context.load("train_target")
+    train_descriptors = settings.context.load("train_descriptors")
+    test_target = settings.context.load("test_target")
+    test_descriptors = settings.context.load("test_descriptors")
+    return train_target, train_descriptors, test_target, test_descriptors
+
+
+class TestPreProcessingScalerFlavors(BaseUnitTest):
+    """
+    This class performs unittests for the pre_processing scaler flavors.
+    We may only check regression data here, and assume it would work for the other data categories.
+    """
+
+    def min_max_pass_condition(self, data):
+        for col in data.T:
+            self.assertAlmostEqual(1.0, np.amax(col))
+            self.assertAlmostEqual(0.0, np.amin(col))
```

Comment:
If we're going to have different pass conditions and tests for different scalers, why not just make a separate class for each scaler? Unless there's a good reason these have to be clumped in the same object.

Reply:
I found it easier to keep these here for now, to avoid having to make new files for inheritance to separate out the common parts.